pupil-spa/src/app/services/audit/auditEntry.ts

```diff
@@ -1,6 +1,6 @@
-export type AuditEntryType = 'WarmupIntroRendered' | 'WarmupCompleteRendered' | 'CheckComplete' |
-'CheckStarted' | 'QuestionRendered' | 'QuestionAnswered' | 'PauseRendered' | 'RefreshDetected' |
-  'UtteranceStarted' | 'UtteranceEnded';
+export type AuditEntryType = 'WarmupIntroRendered' | 'WarmupCompleteRendered' | 'CheckStartedAPICallSucceed' |
```

Comment:
i think we should audit the attempt of the call `CheckStartedApiCalled' and the result.  So i would go with...

`CheckStartedApiCalled`
`CheckStartedApiCallSucceeded`
`CheckStartedApiCallFailed`

Reply:
@GuyHarwood updated